packages/webpack-config-terra/webpack.config.js

```diff
@@ -0,0 +1,256 @@
+const Autoprefixer = require('autoprefixer');
+const PostCSSAssetsPlugin = require('postcss-assets-webpack-plugin');
+const PostCSSCustomProperties = require('postcss-custom-properties');
+const path = require('path');
+const rtl = require('postcss-rtl');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const TerserPlugin = require('terser-webpack-plugin');
+const webpack = require('webpack');
+const merge = require('webpack-merge');
+const DuplicatePackageCheckerPlugin = require('@cerner/duplicate-package-checker-webpack-plugin');
+const aggregateTranslations = require('terra-aggregate-translations');
```

Comment:
oof..curious if we will be having a MVB or not with the react-intl uplift if the local data is not the same: https://david-dm.org/cerner/terra-aggregate-translations?type=peer

Reply:
Maybe? depends if aggregate translations can support multiple major versions of react intl. Unfortunately this package has several major versions in it's future webpack 5, removing aggregate translations and maybe react-intl.